### PR TITLE
Migrating the Wiki documentation to github

### PR DESCRIPTION
**Pull request by RohitMazumder**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+# CHANGELOG
+
+### 3.0-alpha
```
Comment (zbynek) [inline CHANGELOG.md:3]: nitpick: it might be more consistent to change the heading level for all versions from 3 to 2

File: README.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+-   [Persistent cross-site scripting
+    vulnerability](https://jenkins.io/security/advisory/2017-04-10/)
+-   [Cross-site request forgery vulnerabilities in Scriptler script
+    management](https://jenkins.io/security/advisory/2017-04-10/)
+-   [Any user can add Scriptler scripts to build
+    configurations](https://jenkins.io/security/advisory/2017-04-10/)
+-   [Arbitrary code execution vulnerability in rare
+    circumstances](https://jenkins.io/security/advisory/2017-04-10/)
+-   [Any Scriptler script can be executed as part of
+    builds](https://jenkins.io/security/advisory/2017-04-10/)
```
Comment (zbynek) [inline README.md:13]: This is automatically generated text that was inserted to the wiki: the plugin site will fetch the latest security warnings automatically. Please remove.

Some level1 heading like "Scriptler Plugin for Jenkins" could be useful instead.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+how to access the repo via git.
+
+This allows an administrator to push/pull scripts from its
+workstation/IDE into Scriptler. If such a script is pushed the first
+time into scriptler, then it will parse the optional header info like
+this:
+
+**optional script header**
+
+``` syntaxhighlighter-pre
```
Comment (zbynek) [inline README.md:109]:
```suggestion
``` groovy
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+**macro**
+
+``` syntaxhighlighter-pre
+${SCRIPTLER, scriptId="superscript.groovy"}
+```
+
+**superscript.groovy**
+
+``` syntaxhighlighter-pre
```
Comment (zbynek) [inline README.md:145]:
```suggestion
``` groovy
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+2.  as a producer, scriptler exposes all scripts marked as non admin
+    scripts via a macro:  
+    - the value returned by the script is the value returned by the
+    TokenMacro, it is NOT enough to write something to the console/log  
+    - scriptler exposes the current '*build'* to the script executed
+    (since 1.7)
+
+**macro**
+
+``` syntaxhighlighter-pre
```
Comment (zbynek) [inline README.md:139]: also groovy? or just remove `syntaxhighlighter-pre`?

**Review by oleg-nenashev — APPROVED** [review]
I think this is a good start. I will merge it so that more documentation patches can be contributed on the top of it

**Pull request merged**